app/forms/steps/details/documents_checklist_form.rb

```diff
@@ -0,0 +1,35 @@
+module Steps::Details
+  class DocumentsChecklistForm < BaseForm
+    attribute :original_notice_provided, Boolean
+    attribute :review_conclusion_provided, Boolean
+    attribute :additional_documents_provided, Boolean
+    attribute :additional_documents_info, String
+    attribute :having_problems_uploading_documents, Boolean
+
+    validates_presence_of :original_notice_provided,
```

Comment:
Right now, I understand what each of these validations means, and how they fit together. But, when I look at this again, in two months (or whenever), I think it would take a while to figure out.

Should we wrap this up in a method, where we can perhaps communicate the intention more clearly?